src/proto/grpc/lb/v1/load_balancer.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 gRPC authors.
+// Copyright 2015 The gRPC Authors
```

Comment:
I'm just curious about why this gets decreased?

Reply:
The earlier copyright takes precedence.  This file was copied a few times, but the original date holds.